### PR TITLE
(enhancement) bump styling to carbon 11 for form-entry engine

### PR DESCRIPTION
**Pull request by donaldkibet**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.

## Summary

- Update the styling of form-entry to Carbon 11

## Screenshots

*None.*

## Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Review by makombe — APPROVED** [review]
LGTM

**Review by makombe — APPROVED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
         with:
           node-version: "16"
           cache: "npm"
-      - run: yarn install --frozen-lockfile
+      - run: yarn install
```
Comment (ibacher) [inline .github/workflows/main.yml:39]:
```suggestion
      - run: yarn install --frozen-lockfile
```

File: .gitignore [hunk 2]
```diff
@@ -16,6 +16,7 @@
 *.launch
 .settings/
 *.sublime-workspace
+.yarn
```
Comment (ibacher) [inline .gitignore:19]:
```suggestion
```

File: package.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
     "e2e": "ng e2e"
   },
   "peerDependencies": {
-    "carbon-components": "10.x"
+    "@carbon/styles": "1.x"
```
Comment (ibacher) [inline package.json:16]: This should be a dependency, not a peer dependency (it isn't provided at runtime).

File: .gitignore (hunk 2 above)
Comment (ibacher) [reply .gitignore:19]: If you're moving this repo over to Yarn 3, please see the Yarn docs for what needs to be in the .gitignore.

File: src/styles.scss [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/* You can add global styles to this file, and also import other style files */
+@import '~@carbon/styles/css/styles.min.css';
```
Comment (ibacher) [inline src/styles.scss:2]:
```suggestion
@import '~@carbon/styles';
```

File: .github/workflows/main.yml (hunk 1 above)
Comment (ibacher) [inline .github/workflows/main.yml:39]:
```suggestion
      - run: yarn install --frozen-lockfile
```

Or for Yarn v3:

```suggestion
      - run: yarn install --immutable
```

**Review by ibacher — CHANGES_REQUESTED** [review]
@donaldkibet Thanks for this! Some nit-picky changes are necessary, but broadly this looks good.

File: .github/workflows/main.yml (hunk 1 above)
Comment (ibacher) [reply .github/workflows/main.yml]: The yarn lock files need to be updated, but that should be part of your commit. By the time CI is running, the yarn lock file shouldn't need to be updated (this is basically the use-case for this flag).

**Review by denniskigen — APPROVED** [review]

**Review by ibacher — APPROVED** [review]

**Comment by ibacher** [comment]
Thanks @donaldkibet!

**Pull request merged**